charts/atlantis/README.md

```diff
@@ -183,6 +183,7 @@ extraManifests:
 | replicaCount | int | `1` | Replica count for Atlantis pods. |
 | repoConfig | string | `""` | Use Server Side Repo Config, ref: https://www.runatlantis.io/docs/server-side-repo-config.html. Check values.yaml for examples. |
 | resources | object | `{}` | Resources for Atlantis. Check values.yaml for examples. |
+| secretAnnotations | object | `{}` | These annotations will be added to secrets. Check values.yaml for examples. |
```

Comment:
What do you think of using `secret.Annotations` instead? I'm wondering if we will need more `secret` properties and then we just add them. E.g. `labels`.